src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  // ({let partner = person.sex === 'f' ? 'husband' : 'wife'})
+
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age !== undefined && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {person.isMarried === true
```

Comment:
```suggestion
      {person.isMarried
```
no need to compare with true